### ISSUE TITLE
Plant hitbox water collision

### ISSUE THREAD

**Issue by srnyx** [Hephaestus-Dev/TinyTweaks#74]
Uhhhh... I don't think this is right...
_I'm **almost** positive it's because of TinyTweaks_
![image](https://cdn.tixte.com/file/tixtehost/uploads/max-is.an-idiot.xyz/kmv0jihgl09.png)

**Comment by Haven-King** [comment]
I'm not exactly sure what I'm looking at. Could you describe what's happening? My first instinct is that it doesn't seem related to any of TinyTweaks' features, but I could be wrong.

**Comment by srnyx** [comment]
> I'm not exactly sure what I'm looking at. Could you describe what's happening? My first instinct is that it doesn't seem related to any of TinyTweaks' features, but I could be wrong.

The water isn't flowing because the grass is in the way. Usually, the water would destroy the grass.

**Comment by Haven-King** [comment]
I don't believe TinyTweaks is responsible for this, but I'll look into it. Could you provide your mods list?

**Comment by srnyx** [comment]
> I don't believe TinyTweaks is responsible for this, but I'll look into it. Could you provide your mods list?

It may be CutClean
![image](https://user-images.githubusercontent.com/25808801/112893855-30f27180-90a9-11eb-9573-b564a500662c.png)
![image](https://user-images.githubusercontent.com/25808801/112893893-3c459d00-90a9-11eb-995e-e9d0aecc0f65.png)
![image](https://user-images.githubusercontent.com/25808801/112893932-45cf0500-90a9-11eb-96db-44c26e5323c7.png)
![image](https://user-images.githubusercontent.com/25808801/112893979-52535d80-90a9-11eb-908f-b237448c0dd7.png)
